src/docfx/config/Config.cs

```diff
@@ -49,12 +49,12 @@ internal class Config
         /// <summary>
         /// Just for backward compatibility, the output site path prefix
         /// </summary>
-        public readonly string SiteBasePath = string.Empty;
+        public readonly string SiteBasePath = ".";
 
         /// <summary>
         /// Just for backward compatibility, the source path prefix
         /// </summary>
-        public readonly string SourceBasePath = string.Empty;
+        public readonly string SourceBasePath = ".";
```

Comment:
>readonly [](start = 15, length = 8)

how about const instead of readonly and we do not need to check empty afterwards? it needed to be readonly since it was string.Empty. Same for "SiteBasePath"

Reply:
it is not a const, just a default value

---
In reply to: [206002546](https://github.com/dotnet/docfx/pull/3120#discussion_r206002546) [](ancestors = 206002546)